### ISSUE TITLE
Horizontal "align center" not works properly on tabbed layout

### ISSUE THREAD

**Issue by uunnxx** [svenstaro/wmfocus#70]
Hi, I can't set it to work properly in the center of the screen.

```wmfocus  --valign center --halign center```

![2020-10-16-185654_3840x1080_scrot](https://user-images.githubusercontent.com/6236057/96281296-d1644600-0fe1-11eb-978e-addd80e35c8c.png)

I thought it's because of tabbed layout, but when I set --halign left or when I remove it, it will work as expected:
```wmfocus --valign center --halign left```
or
```wmfocus --valign center```

![2020-10-16-185944_3840x1080_scrot](https://user-images.githubusercontent.com/6236057/96281402-f2c53200-0fe1-11eb-9d30-c823a438e293.png)

```wmfocus --version # => 1.1.4```

System information:
```
uname -a
Linux host 5.4.0-47-generic #51-Ubuntu SMP Fri Sep 4 19:50:52 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by svenstaro** [comment]
Hm could you sketch our what you think it should look like?

**Comment by uunnxx** [comment]
RED to GREEN

```wmfocus --valign center --halign center```

![centered but need to left](https://user-images.githubusercontent.com/6236057/96288986-f1e5cd80-0fec-11eb-8d12-f37d8bda4df6.jpg)

as an example when we set
```wmfocus --valign center --halign left```
or
```wmfocus --valign center```
in the --halign center it should look  like this

![its lefted but we need center](https://user-images.githubusercontent.com/6236057/96289092-1e99e500-0fed-11eb-848a-982ed2daf0f1.jpg)

**Comment by svenstaro** [comment]
That's actually very helpful. I'll see about fixing this but feel free to take a stab at this if you like. :)

**Comment by svenstaro** [comment]
This should now be fixed. Please check. Released as 1.1.5.
